### PR TITLE
feact : added profile page

### PR DESCRIPTION
**Pull request by anupkgurung**
Added page for profile
Edit profile (bio & web link)
follow & unfollow user

### CODE REVIEW

**Review by vermasavita — APPROVED** [review]
All functionalities are working fine. Good to merge in my opinion.

File: src/component/Modal/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <div className="relative">
+                    <img className="rounded-full border-2 border-white w-20 h-20" src="https://raw.githubusercontent.com/anupkgurung/images/main/social/anup.jpg" alt="anupkmr" />
+                    <span className="material-icons-outlined absolute object-cover rounded-full cursor-pointer top-[3.5rem] left-[3rem] bg-green-200">add_a_photo</span>
+                </div>
+                <div>{user?.username}</div>
+              
+                <div className="flex flex-col w-full h-16 ">
+                    <span className="mr-auto">PORTFOLIO</span>
+                    <input id="portfolio" type="text" className="w-full h-8 border rounded border-blue-400 outline-blue-400" value={userData.portfolio} 
+                        onChange={changeHanlder}
```
Comment (MarkVed17) [inline src/component/Modal/Modal.jsx:34]: Handler*

File: src/component/Modal/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                    />
+                </div>
+                <div className="flex flex-col w-full my-2 h-28">
+                    <span className="mr-auto">BIO</span>
+                    <textarea  id="bio" type="text" className="border-blue-400 border h-28 w-full rounded outline-blue-400" value={userData.bio} 
+                        onChange={changeHanlder}
```
Comment (MarkVed17) [inline src/component/Modal/Modal.jsx:40]: here as well

**Review by MarkVed17 — APPROVED** [review]
Liked the naming conventions. Makes it easier to read the code.
There are a few suggestions:

- Expect some space between these two buttons:
image

- Need an `href` for external links. For your github profile. As you may observe that clicking on this won't redirect you to github profile.
image

- App breaks on clicking the `Post` button

**Pull request merged**